.github/actions/generate/action.yml

```diff
@@ -12,6 +12,14 @@ inputs:
     description: Build type for third-party libraries and clio. Could be 'Release', 'Debug'
     required: true
     default: 'Release'
+  build_integration_tests:
+    description: Whether to build integration tests
+    required: true
+    default: 'true'
+  lint:
+    description: Whether to build with linter enabled
+    required: true
+    default: 'false'  
```

Comment:
I don't think we need this one. Linting during compilation might be useful for development, but because build fails on the first issue `clang-tidy` found, I think it is not useful in CI.